sdk/communication/Azure.Communication.MediaComposition/src/autorest.md

```diff
@@ -0,0 +1,17 @@
+# Azure.Communication.MediaComposition
+
+Run `dotnet msbuild /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+title: Media Composition
+tag: package-2022-06-26-preview1
+generation1-convenience-client: true
```

Comment:
ah, interesting. Is that the new flag to opt-into the behavior before Dataplane-Generation?

Reply:
Actually, I found this flag in another ACS SDK and I had to include it for the models to be generated. I'm not sure exactly what it's meant to do but was required for autorest to work properly